Packs/Cisco-umbrella/Integrations/Cisco-umbrella-investigate/Cisco-umbrella-investigate.py

```diff
@@ -748,113 +750,128 @@ def get_domain_command():
             }
-            dbotscore = 3
 
-        context[outputPaths['dbotscore']] = {
-            'Indicator': domain,
-            'Type': 'domain',
-            'Vendor': 'Cisco Umbrella Investigate',
-            'Score': dbotscore,
-            'Reliability': reliability
-        }
-
-        contents.append({
-            'Risk Score': risk_score,
-            'Secure Rank': secure_rank,
-            'Populairty': popularity,
-            'Demisto Reputation': scoreToReputation(dbotscore),
-            'First Queried time': first_queried,
-        })
-
-        # Domain reputation + [whois -> whois nameservers -> whois emails] + domain categorization
-        results.append({
-            'Type': entryTypes['note'],
-            'ContentsFormat': formats['json'],
-            'Contents': [contents, whois, name_servers, emails, domain_categorization_table],
-            'ReadableContentsFormat': formats['markdown'],
-            'HumanReadable':
-                tableToMarkdown('"Umbrella Investigate" Domain Reputation for: ' + domain, contents, headers)
-                + tableToMarkdown('"Umbrella Investigate" WHOIS Record Data for: ' + domain, whois, headers,
-                                  date_fields=["Last Retrieved"])
-                + tableToMarkdown('Name Servers:', {'Name Servers': name_servers}, headers)
-                + tableToMarkdown('Emails:', emails, ['Emails'])
-                + tableToMarkdown('Domain Categorization:', domain_categorization_table, headers),
-            'EntryContext': context
-        })
+            contents.append({
+                'Risk Score': risk_score,
+                'Secure Rank': secure_rank,
+                'Populairty': popularity,
+                'Demisto Reputation': scoreToReputation(dbotscore),
+                'First Queried time': first_queried,
+            })
 
+            # Domain reputation + [whois -> whois nameservers -> whois emails] + domain categorization
+            results.append({
+                'Type': entryTypes['note'],
+                'ContentsFormat': formats['json'],
+                'Contents': [contents, whois, name_servers, emails, domain_categorization_table],
+                'ReadableContentsFormat': formats['markdown'],
+                'HumanReadable':
+                    tableToMarkdown('"Umbrella Investigate" Domain Reputation for: ' + domain, contents, headers)
+                    + tableToMarkdown('"Umbrella Investigate" WHOIS Record Data for: ' + domain, whois, headers,
+                                      date_fields=["Last Retrieved"])
+                    + tableToMarkdown('Name Servers:', {'Name Servers': name_servers}, headers)
+                    + tableToMarkdown('Emails:', emails, ['Emails'])
+                    + tableToMarkdown('Domain Categorization:', domain_categorization_table, headers),
+                'EntryContext': context
+            })
+        except RequestException as r:
+            if '404' in r.message:
```

Comment:
why do we check it inside the message and not using the response code?

Reply:
No reason, changed it to check the status code of the response